roles/partition-disk/tasks/remove_home_partition.yaml

```diff
@@ -0,0 +1,18 @@
+---
+  - name: "Unmount /home partition"
+    mount:
+      name: /home
+      state: unmounted
+  - name: "Remove /home entry from /etc/fstab"
+    mount:
+      name: /home
+      state: absent
+  - name: "Remove /home Logical Volume"
```

Comment:
I find it helps readability to separate tasks by an empty line just like you would a function definition in python code